.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: BuildStockBatch Tests
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    name: Tests - Python ${{ matrix.python-version }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install buildstockbatch
+        run: |
+          python -m pip install --progress-bar off --upgrade pip
+          pip install .[dev] --progress-bar off
+      - name: Linting
```

Comment:
That's interesting. I haven't done that. I actually want to switch our linting/formatting to [black](https://github.com/psf/black) but was going to do that as its own PR once a few more things have merged down because it's going to touch _a lot_ of lines of code.